bundler-audit.gemspec

```diff
@@ -22,7 +22,8 @@ Gem::Specification.new do |gem|
   glob = lambda { |patterns| gem.files & Dir[*patterns] }
 
   gem.files = if gemspec['files'] then glob[gemspec['files']]
-              else                     `git ls-files`.split($/)
+              else
+                `git ls-files`.split($/)
```

Comment:
I actually prefer the `if ... then ...` syntax. If the `else` clause is going to be on the next line, then I feel like the same should be done to the `if` clause. Or, you could just disable that rubocop rule.